src/libs/CurrencyUtils.ts

```diff
@@ -49,8 +49,8 @@ function getCurrencyUnit(currency: string = CONST.CURRENCY.USD): number {
 /**
  * Get localized currency symbol for currency(ISO 4217) Code
  */
-function getLocalizedCurrencySymbol(currencyCode: string): string | undefined {
-    const parts = formatToParts(IntlStore.getCurrentLocale(), 0, {
+function getLocalizedCurrencySymbol(locale: Locale | undefined, currencyCode: string): string | undefined {
+    const parts = formatToParts(locale, 0, {
```

Comment:
This now changes the logic of this function. Earlier, we were looking at user's system Locale. Now it is checking the preference. What is the reason for this change?